src/api/endpoints/analytics.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import APIRouter, Depends
+
+from src.api.schemas import Analytic
+from src.api.services.analitics import AnalyticsService
+
+analytic_router = APIRouter()
+
+
+@analytic_router.get("/", response_model=Analytic, description="Возращает статистику сервиса.")
+async def get_statistics(analytic_service: AnalyticsService = Depends()) -> Analytic:
+    return await analytic_service.get_user_number()
```

Comment:
Как я уже говорил, функция `get_staticstics` должна возращать статистику, сейчас она возращает число... 
```suggestion
    return Analytic(
        users_number=await analytic_service.get_user_number(),
    )
```